new-frontiers-14/frontier-station-14
# Adds chem lab to NFSD HQ

## About the PR



- Some slight changes to the interior arrangement of the NFSD HQ POI thing. New vents/firelocks plugged in.
- Rebuilds the medical bay with a basic equipment setup.
- Removes the ATM space to make room for a few additional beds. ATM moved to the lobby.
- All should be locked behind the same medical door present on the Empress.
- Replaces empty Brigmed locker with a Brigmed locker that has equipment in it. To remove the need to pull an Empress. 

## Why / Balance


- Recent discussion regarding Brigmeds relying on an Empress being pulled, more or less, and how that's not universally desirable resulted in some of those tools being added to their clubhouse.
- ATM moved to lobby after discussing the matter with a number of sheriffs who urged that it was not only fine but be appreciated. I think they know better than I do. 
- Nobody could explain why the brigmed locker was empty. Only that they needed to pull the ship we're trying to avoid needing to pull to get their stuff. So. 

## How to test


Visit the NFSD HQ. Check security access. Check tiling. Check to make sure room is functional and accessible. 

## Media


![ss+(2024-12-18+at+06 36 34)](https://github.com/user-attachments/assets/46e8b771-da2c-47dd-9be1-e31014ce7cc1)


## Requirements



## Breaking changes


**Changelog**


🆑 
- tweak: Add chem lab to NFSD HQ.
